BHoM_UI/Templates/Caller.cs

```diff
@@ -241,22 +244,47 @@ public virtual bool Read(string json)
                 // New serialisation, we stored a CustomObject with SelectedItem, InputParams and OutputParams
                 object backendElement;
                 if (component.CustomData.TryGetValue("SelectedItem", out backendElement))
+                {
+                    if (backendElement == null)
+                    {
+                        BsonDocument bson = Engine.Serialiser.Convert.ToBson(json);
```

Comment:
I don't think this should be handled here. We have the `Versioning_Engine`, right?

Reply:
I think that's been added by @adecler's changes.